dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGVisualizer.java

```diff
@@ -132,6 +132,15 @@
      */
     String visualizeStoreKeyVal(String keyName, Object value);
 
+    /**
+     * Visualize specific information collected according to the specific kind of Store. Currently,
+     * {@code LiveVariableStore} uses this method.
+     *
+     * @param value the value of the specific information to be visualized
+     * @return the String representation of the store
+     */
+    String visualizeStoreVal(Object value);
```

Comment:
It's not really clear what this method is for. Is this just to get quoting and escaping of the string right?
Can you think of a better name and then maybe use the method more widely?